src/backend/scripts/setup_localdata.py

```diff
@@ -60,6 +60,7 @@ def create_test_user(session, group, user_id, name):
         group_admin=True,
         system_admin=True,
         group=group,
+        agreed_to_tos=True,  # TODO - FE tests need to be updated for agree modal!
```

Comment:
This is a bit of a hack at the moment -- this disables the 'agree to tos' modal that was breaking most of our tests. I think it would be nicer if our tests could detect whether the modal exists and click ok if necessary, but my TS isn't quite up to the task right now.

Reply:
Yeah that should definitely be doable! Looking forward to your future TS adventures